examples/chicane/input_chicane.in

```diff
@@ -26,15 +26,15 @@ lattice.elements = monitor sbend1 dipedge1 drift1 dipedge2 sbend2 drift2      \
 lattice.nslice = 25
 
 sbend1.type = sbend
-sbend1.ds = 0.50037       # projected length 0.5 m, angle 2.77 deg
-sbend1.rc = -10.35
+sbend1.ds = 0.500194828041958       # projected length 0.5 m, angle 2.77 deg
+sbend1.rc = -10.3462283686195526
 
 drift1.type = drift
 drift1.ds = 5.0058489435  # projected length 5 m
 
 sbend2.type = sbend
-sbend2.ds = 0.50037       # projected length 0.5 m, angle 2.77 deg
-sbend2.rc = 10.35
+sbend2.ds = 0.500194828041958       # projected length 0.5 m, angle 2.77 deg
+sbend2.rc = 10.3462283686195526
```

Comment:
If you want, we could use user-defined constants and the parser here to make it a bit clearer how those (projected) parameters come to play.
- https://impactx.readthedocs.io/en/latest/usage/parameters.html#running-cpp-parameters-parser